tests/load_data/event_data/test_opta.py

```diff
@@ -0,0 +1,190 @@
+                "minutes": [0, 0, 0, 0, 0, 0, 30, 30, 31],
+                "seconds": [0, 0, 0, 1, 4, 6, 9, 10, 10],
+                "player_id": [
+                    np.nan,
+                    np.nan,
+                    np.nan,
+                    181078,
+                    510654,
+                    510654,
+                    223089,
+                    510654,
+                    223089,
+                ],
+                "team_id": [425, 318, 425, 318, 318, 318, 425, 318, 425],
+                "outcome": [1, 1, 1, 1, 0, 0, 1, 0, 1],
+                # field dimensions should be [10, 10] to scale down all values by
+                # a factor of 10
+                "start_x": [5.0, -5.0, 5.0, -0.03, -1.84, -1.9, 5.0, 1.57, 1.57],
+                "start_y": [5.0, -5.0, 5.0, 0.01, -0.93, -0.57, 5.0, -2.68, -2.68],
+                "datetime": np.array(
+                    [
+                        "2023-01-22T10:28:32.117",
+                        "2023-01-22T11:18:32.152",
+                        "2023-01-22T11:18:32.152",
+                        "2023-01-22T11:18:33.637",
+                        "2023-01-22T11:18:36.207",
+                        "2023-01-22T11:18:39.109",
+                        "2023-01-22T11:18:41.615",
+                        "2023-01-22T11:18:43.119",
+                        "2023-01-22T11:18:43.120",
+                    ],
+                    dtype="datetime64",
+                ),
+                "player_name": [
+                    np.nan,
+                    np.nan,
+                    np.nan,
+                    "Anas Tahiri",
+                    "Timo Zaal",
+                    "Timo Zaal",
+                    "Johan Hove",
+                    "Timo Zaal",
+                    "Johan Hove",
+                ],
+            }
+        )
+
+    def test_load_opta_event_data(self):
+
+        metadata, event_data = load_opta_event_data(
+            self.f7_loc, self.f24_loc, pitch_dimensions=[10, 10]
+        )
+        pd.testing.assert_frame_equal(event_data, self.expected_event_data)
+        assert metadata.match_id == self.expected_metadata.match_id
+
+    def test_load_metadata(self):
+
+        metadata = _load_metadata(self.f7_loc)
+        assert metadata == self.expected_metadata
+
+    def test_get_player_info(self):
+        player_data = [
+            {
+                "PlayerRef": "s123",
+                "Formation_Place": "0",
+                "Position": "Substitute",
+                "SubPosition": "Midfielder",
+                "Status": "Substitute",
+                "ShirtNumber": "33",
+            },
+            {
+                "PlayerRef": "s234",
+                "Formation_Place": "1",
+                "Position": "GoalKeeper",
+                "Status": "Start",
+                "ShirtNumber": "2",
+            },
+        ]
+        player_names = {"123": "Sven Kerhoffs", "234": "Niels Smits"}
+
+        expected_result = pd.DataFrame(
```

Comment:
Waarom staan sommige expected results in setUp en sommige in de dedicated test functies?